ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -59,7 +64,8 @@ Create Model Server
     ELSE IF    ${token}==${TRUE}
         Enable Token Authentication
     END
-    Click Button    Add
```

Comment:
have you seen this failing without the wait?

Reply:
yes, it happened. That's why making sure the the "Add" button is enabled before pressing it.